lms/services/group_info.py

```diff
@@ -35,9 +35,12 @@ def upsert_group_info(self, grouping: Grouping, params: dict):
         if not group_info:
             group_info = GroupInfo(
                 authority_provided_id=grouping.authority_provided_id,
-                application_instance=grouping.application_instance,
             )
             self._db.add(group_info)
+            # SQLA not happy if added directly ot the object before adding it to the session
```

Comment:
This one I can't get to the bottom of. Not sure if it's a problem with the testing setup or the actual code.